Source/GenerateUserTypesFromPdb/Program.cs

```diff
@@ -611,8 +618,8 @@ static void Main(string[] args)
                     compilerParameters.ReferencedAssemblies.Add(MicrosoftCSharpDll);
                 }
 
-                compilerParameters.ReferencedAssemblies.Add(Path.Combine(binFolder, "CsDebugScript.Engine.dll"));
-                compilerParameters.ReferencedAssemblies.Add(Path.Combine(binFolder, "CsDebugScript.CommonUserTypes.dll"));
+                compilerParameters.ReferencedAssemblies.Add(Path.Combine(binFolder, "CsDebugScript.dll"));
```

Comment:
Renamed. I think this should be taken from the config.
But I do not test this path.